website/blog/2024-04-22-extended-attributes.md

```diff
@@ -0,0 +1,103 @@
+
+## How Extended Attributes Work
+
+Extended Attributes are a textbox on the Environment settings page, where you can input `profiles.yml` type configurations. When developing in the dbt Cloud IDE, dbt Cloud CLI, or orchestrating job runs, dbt Cloud will parse through the provided Extended Attributes YAML and merge it with your base project connection settings. If the attribute exists in another source (at present this would typically be your project connection settings or the Job's configurations), it will _replace_ its value, including overriding any custom environment variables. If the attribute doesn't exist, it will add the attribute to the connection config. You [can check out the documentation](https://docs.getdbt.com/docs/deploy/deploy-environments#extended-attributes) for more specific details, but now that you’ve got the basic idea, let’s dive into some examples to see why this is so cool.
+
+## Multiple accounts for Development and Production environments
+
+The most pressing use case for dbt Cloud users is the ability to use different account connections for different teams or development stages in their pipelines. Let’s consider a team that has a typical dev, staging, production setup (known as a Write-Audit-Publish [WAP] workflow): development for active work with small datasets, staging to promote and vet changes against cloned production data, and production for the final deployed code that feeds BI tools. For this team though, these are separate _accounts_ in their data platform with their own sets of RBAC and other settings. This is a perfect use case for extended attributes. Let’s take a look at how they might set this up for a company that uses multiple BigQuery accounts, projects, and datasets (projects and datasets are analogous to databases and schemas on other platforms like Snowflake) to separate Dev, Staging, and Prod:
+
+![Extended Attributes text box](/img/blog/2024-04-10-extended-attributes/ext_attr.png)
+
+_<small>At the bottom of an Environment's Settings you’ll find the Extended Attributes text box.</small>_
+
+### Development
+
+```yaml
+account: 123dev
+project: dev
+dataset: dbt_winnie
+method: oauth
+threads: 1
+```
+
+### Staging
+
+```yaml
+account: 123dev
+project: staging
+dataset: main
+method: service-account-json
+threads: 16
+```
+
+### Production
+
+```yaml
+account: 456prod
+project: analytics
+dataset: main
+method: service-account-json
+threads: 16
+```
+
+With this setup, we have a separate account for development work, using individual development datasets for each developer (with a single thread so that the development build logs are easier to read) connected via OAuth, and a shared `staging` project with a default `main` dataset for the Staging environment that is only built to via a GCP Service Account through dbt Cloud. In that project we can then configure IAM permissions to only allow building into the Staging schema from jobs that use the Staging environment as well.
```

Comment:
sugg: since semicolons can be used to list things, sugg using it here for clarity 

```suggestion
With this setup, we have a separate account for development work, using individual development datasets for each developer (with a single thread so that the development build logs are easier to read) connected via OAuth; and a shared `staging` project with a default `main` dataset for the staging environment that's only built via a GCP Service Account through dbt Cloud. In that project, we can then configure IAM permissions to only allow building into the staging schema from jobs that use the staging environment as well.
```